### ISSUE TITLE
Option to use this plugin without actual formation of the hard links

### ISSUE THREAD

**Issue by tallguyjenks** [aviskase/obsidian-link-indexer#9]
So when this plugin is run and generates the output file it actually links to all the files in the vault and makes the graph revolve around a single file:

![image](https://user-images.githubusercontent.com/29872822/99289723-2e763600-27f2-11eb-8bf6-cb5c80d652bc.png)

The hack i was using before this plugin came out was a command line one-liner to build this output file:

```bash
egrep -ohsr --exclude-dir='.*' "\[\[[A-Za-z0-9 _@ÄÖÜäöüß\'\!\?\.\(\)\-]+\]\]" -- * | sed -e 's/[[:space:]]#/#/' | sort | uniq -c | sort -r -t# -k2  > "🔗️☁️.md"
```

and it generates a file that has no line breaks between links (your template produces line breaks between links) and also ignored all daily notes based on the date format (another great feature to add)

the output file was also indented so that it doesn't actually create links and distort the graph:

![image](https://user-images.githubusercontent.com/29872822/99289987-9298fa00-27f2-11eb-997b-03704d249288.png)

in obsidian's settings, you can turn off strict line breaks so

```
[[link1]]
[[link2]]
```

appear exactly as they are seen above and not like:

`[[link1]][[link2]]`

but this can be toggled, the request would be to toggle the plugin so you can have the output displayed like 

```
[[link1]]
[[link2]]
```

OR

```
[[link1]]

[[link2]]
```

if you have those line breaks turned on.

![image](https://user-images.githubusercontent.com/29872822/99299080-89faf080-27ff-11eb-8430-274f10e696e8.png)

just some ideas :)

**Comment by aviskase** [comment]
@tallguyjenks great ideas, so, to summarize:

- toggle to have use strict line breaks or not
- toggle to have link `[[file_link]]` vs plaintext `file_link`
- ignore paths with glob 

Did I catch all?

**Comment by tallguyjenks** [comment]
1. toggle to have use strict line breaks or not
2. toggle to have link [[file_link]] vs plaintext file_link
3. ignore paths with glob

i actually didn't care about 2, but that's a good idea! Yes!

**Comment by aviskase** [comment]
(2) solves the same problem as indenting, but I think will be less confusing. Thanks, gonna work on these first!

**Comment by aviskase** [comment]
(1) and (2) are implemented, (3) is logged as a separate issue #11
